src/silx/math/colormap.py

```diff
@@ -449,16 +456,12 @@ def _get_normalizer(norm: str, gamma: float) -> _NormalizationMixIn:
     return _BASIC_NORMALIZATIONS[norm]
 
 
-def _get_range(
-    normalizer,
-    data: numpy.ndarray,
-    autoscale: AutoScaleModeType,
-    vmin: float | None,
-    vmax: float | None,
-) -> tuple[float, float]:
+def _get_range(normalizer, data, autoscale, vmin, vmax, saturation: float = 0.0):
```

Comment:
Can you take added typing into account?

Reply:
I really don't get how I could have miss this one. Did you did some rebase at one point ?

Reply:
In between we have merged stuff in the main branch.

You just have to rebase i think. You will have conflicts.